### PR TITLE
Parse additional eICR fields (race, ethnicity, DOB, gender)

### PR DESCRIPTION
**Pull request by DanPaseltiner**
# PULL REQUEST

## Summary
This PR adds DOB, race, ethnicity, and gender to the ECR parsing schema in the message parser so that this data can be extracted from eICRs.

## Additional Information
Anything else the review team should know?

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/message-parser/app/default_schemas/ecr.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
   "person_id": "Bundle.entry.resource.where(resourceType='Person').id",
   "last_name": "Bundle.entry.resource.where(resourceType = 'Patient').name.first().family",
   "first_name": "Bundle.entry.resource.where(resourceType = 'Patient').name.first().given.first()",
+  "birth_date": "Bundle.entry.resource.where(resourceType = 'Patient').birthDate",
+  "gender_identity": "Bundle.entry.resource.where(resourceType = 'Patient').gender",
+  "race": "Bundle.entry.resource.where(resourceType = 'Patient').extension.where(url = 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-race').extension.valueCoding.display",
+  "ethnicity": "Bundle.entry.resource.where(resourceType = 'Patient').extension.where(url = 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity').extension.valueCoding.display",
```
Comment (BradySkylight) [inline containers/message-parser/app/default_schemas/ecr.json:9]: Nice FHIR Path to get those extension values.  Is gender the same as gender_identity?  I was wondering if there was a separate field for that in the XML EICR.

File: containers/message-parser/tests/test_parse_message_endpoint.py [hunk 2]
```diff
@@ -18,227 +18,35 @@
...
-        "test_type_code_20": "",
-        "test_result_20": "",
-        "test_result_interp_20": "",
-        "specimen_type_20": "",
-        "performing_lab_20": "",
-        "specimen_collection_date_20": "",
-        "result_date_20": "",
-    },
-}
-
```
Comment (BradySkylight) [inline containers/message-parser/tests/test_parse_message_endpoint.py:219]: Thanks for cleaning these tests up!

File: containers/message-parser/app/default_schemas/ecr.json (hunk 1 above)
Comment (DanPaseltiner) [reply containers/message-parser/app/default_schemas/ecr.json:9]: Hmm good question. Maybe we just call this simply "gender" for now?

Comment (DanPaseltiner) [reply containers/message-parser/app/default_schemas/ecr.json:9]: changed to gender.

Comment (BradySkylight) [reply containers/message-parser/app/default_schemas/ecr.json:9]: As discussed you will change to gender as it's being pulled from administrativeGender within the EICR XML (CDA).

**Review by BradySkylight — APPROVED** [review]
LGTM!

**Pull request merged**